### PR TITLE
Fixed where the keplr overides txn fees.

### PR DESCRIPTION
**Pull request by klk1236**

### CODE REVIEW

**Review by webmaster128 — COMMENTED** [review]
I am not familiar with Keplr configs but seems plausible.

@nooomski @samepant maybe you can verify this?

File: components/forms/TransactionSigning.tsx [hunk 1]
```diff
@@ -33,6 +33,9 @@ const TransactionSigning = (props: Props) => {
     try {
       assert(state.chain.chainId, "chainId missing");
       await window.keplr.enable(state.chain.chainId);
+      window.keplr.defaultOptions = {
+        sign: { preferNoSetFee: !0, preferNoSetMemo: !0, disableBalanceCheck: !0 },
```
[the remark this answers is not in the document]
Comment (puneet2019) [reply components/forms/TransactionSigning.tsx:37]: I just copied it from some code I guess. :|

[the remark this answers is not in the document]
Comment (klk1236) [reply components/forms/TransactionSigning.tsx]: I have set it to true, Works the same.

**Review by samepant — APPROVED** [review]
tested, this works, thanks for the contrib

**Pull request merged**